src/main/java/com/mifos/utils/PrefManager.java

```diff
@@ -112,6 +113,14 @@ public void putStringSet(String preferencesKey, Set<String> values) {
         return getPreferences().getStringSet(preferencesKey, null);
     }
 
+    public void setAppLabels(AppLabels appLabels) {
```

Comment:
Check with praharsh, he has created method for  get and set of object in prefmanager, you can reuse the same method.